Bio/Data/IUPACData.py

```diff
@@ -53,16 +53,14 @@
     "W": "Trp",
     "Y": "Tyr",
 }
-protein_letters_1to3_extended = dict(
-    list(protein_letters_1to3.items())
-    + list(
-        {"B": "Asx", "X": "Xaa", "Z": "Glx", "J": "Xle", "U": "Sec", "O": "Pyl"}.items()
-    )
-)
+protein_letters_1to3_extended = {
+    **protein_letters_1to3,
+    **{"B": "Asx", "X": "Xaa", "Z": "Glx", "J": "Xle", "U": "Sec", "O": "Pyl"},
+}
```

Comment:
That is some rather concise magic syntax, but nice.

Reply:
Indeed, list and dict unpacking can be used to achieve surprising feats in elegant ways. Like function arguments with illegal variable names.

```python
$ ipython
In [1]: d = {"a key": "a value"}
In [2]: def f(**kwargs): print(kwargs.keys())
In [3]: f(**d)
dict_keys(['a key'])
```